### PR TITLE
Proxy schema

### PR DESCRIPTION
**Pull request by NathanSegerlind**
not tested yet because we need to merge in the changes to set up and deploy onto a cluster w/ proxy data 

but... provide feedback, if people like this approach, I can quickly change dns and flow to use a similar system...

Getting the schema names from an object will put us in a good place for loading the schema from a file common to ingest/ml/oa should we ever go there

### CODE REVIEW

File: src/main/scala/org/opennetworkinsight/proxy/ProxySchema.scala [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  // fields from the input
+
+  val Date = "p_date"
+  val Time = "p_time"
+  val ClientIP = "clientip"
+  val Host = "host"
+  val ReqMethod = "reqmethod"
+  val UserAgent = "useragent"
+  val ResponseContentType = "resconttype"
```
Comment (NathanSegerlind) [inline src/main/scala/org/opennetworkinsight/proxy/ProxySchema.scala:13]: i should look up what the actual full field names are for these things... I just extrapolated this one

**Comment by rabarona** [comment]
I like it.

File: src/main/scala/org/opennetworkinsight/proxy/ProxySchema.scala (hunk 1 above)
Comment (rabarona) [inline src/main/scala/org/opennetworkinsight/proxy/ProxySchema.scala:13]: Are you talking about Duration? 
I'm seeing a few fields called "int" with a space between. Is that the correct column name or is it the name+type?

Comment (NathanSegerlind) [inline src/main/scala/org/opennetworkinsight/proxy/ProxySchema.scala:13]: i got the schema names from the latest setup file... the duration_int thing is just the name plus the type, i think

**Comment by NathanSegerlind** [comment]
we were finally able to ingest data in the new format and run the test... all pipelines are working as expected with this branch and the latest schema

**Pull request merged**